api/swagger-spec/v1.json

```diff
@@ -2826,6 +3126,75 @@
       "consumes": [
        "*/*"
       ]
+     },
+     {
+      "type": "unversioned.Status",
+      "method": "DELETE",
+      "summary": "delete collection of Node",
+      "nickname": "deletecollectionNamespacedNode",
```

Comment:
That's a good catch - although this is generated by our scripts.
This is some more generic issue - see e.g. that Get Node is also saying that node is namespaced:
https://github.com/kubernetes/kubernetes/blob/master/api/swagger-spec/v1.json#L2728

So this is unrelated to this PR.
I filed https://github.com/kubernetes/kubernetes/issues/18497 for it.